### DISCUSSION TITLE
PR with changes/new added files after dependency update

### DISCUSSION THREAD

**Discussion by cklok1997** [renovatebot/renovate#11040] [Request Help] [answered]
Want to know if renovate supports changes/new added files together with lock files in PR?

**Comment by rarkins** [comment, marked as answer]
Yes

**Reply by cklok1997** [reply]
Currently i have files generated from yarn install, but only lock files in the PR. may i know any configuration needed to add in to achieve ?

**Reply by cklok1997** [reply]
Do you have more explanation or reference regarding this discussion? Because my case that generated files is missing in PR, or you need more information from my side?

**Reply by rarkins** [reply]
You have not provided enough information

**Reply by cklok1997** [reply]
I am using self hosted renovate and platform on gitlab.
I run "git status" command in postUpgradeTasks that shown my files "assets-raw" was generated through yarn install under untracked files.
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
        modified:   package.json
        modified:   yarn.lock

Untracked files:
  (use "git add <file>..." to include in what will be committed)
        assets-raw/

no changes added to commit (use "git add" and/or "git commit -a")

<img width="993" alt="Screenshot 2021-08-02 at 12 07 06 AM" src="https://user-images.githubusercontent.com/88034051/127777743-03f2c0fe-bf67-40a2-a10c-41298cbfeb41.png">

Image above show PR missing files my generated files.
Expected result PR should include my new generated files as well.

**Reply by rarkins** [reply]
I was not able to understand that from your previous descriptions. Random files are not committed by default. Run postUpgradeTasks with a dummy command (eg "git status") and then set fileFilters to match the assets you want added. You'll need to enable the command using allowedPostUpgradeCommands

**Reply by cklok1997** [reply]
Problem was solved thank you for your explanation on using postUpgradeTasks
